src/Sarif.FunctionalTests/v2/ConverterTestData/PREfast/Expected-050.xml.sarif

```diff
@@ -101,7 +99,7 @@
         {
           "ruleId": "26000",
           "message": {
-            "text": "Overflow using expression 'a[a[0]]'\nBuffer accessed is a\nBuffer is of length 10 elements (4 bytes/element) [size of variable]\nAccessing the element (4 bytes/element) at element offset 10\n\nValues of variables:\na[0] = 10\n\nThere are other instances of this error:\nPotential read overflow using expression 'a[i]' at line 7\nPotential overflow using expression 'a[a[1]]' at line 11\nOverflow using expression 'a[a[0]]' at line 15\nPotential overflow using expression 'a[a[2]]' at line 18\n"
+            "text": "\n      Overflow using expression 'a[a[0]]'\n            Buffer accessed is a\n            Buffer is of length 10 elements (4 bytes/element) [size of variable]\n            Accessing the element (4 bytes/element) at element offset 10\n      \n            Values of variables:\n            a[0] = 10\n      \n            There are other instances of this error:\n            Potential read overflow using expression 'a[i]' at line 7\n            Potential overflow using expression 'a[a[1]]' at line 11\n            Overflow using expression 'a[a[0]]' at line 15\n            Potential overflow using expression 'a[a[2]]' at line 18\n          "
```

Comment:
>\n       [](start = 21, length = 8)

As a result of your reformatting the PREfast XML files, all the messages have this leading white space (and multiline messages have the white space before each line). Is this worth it? What was the rationale for the reformatting?

Reply:
I mean, I saw your comment that says "VS did it", but still...

---
In reply to: [243852678](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243852678) [](ancestors = 243852678)

Reply:
The truth is that this reformatting wasn't necessary. I happened to reformat just for readability. On doing so, however, I observed we have this undesired introduction of newlines, etc., that flow from the original file to SARIF. I left these artifacts so that we'd drive code improvements. Tracking item below. I suggest deferring all converter improvements until we get past the files array change (when we'll be ready to do more of a wholesale conversion of prerelease sarif v2 to current). Right now we need all the old content to help validate the ongoing prerelease transformation work.

[https://github.com/Microsoft/sarif-sdk/issues/1169](https://github.com/Microsoft/sarif-sdk/issues/1169)

---
In reply to: [243852699](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243852699) [](ancestors = 243852699,243852678)